tst/unit/CMakeLists.txt

```diff
@@ -50,4 +50,7 @@ ParseAndAddCatchTests(unit_tests)
 
 get_property(ALL_TESTS DIRECTORY . PROPERTY TESTS)
 
+foreach(TEST ${ALL_TESTS})
+    set_property(TEST "${TEST}" APPEND PROPERTY LABELS unit)
+endforeach()
```

Comment:
Is this required? If yes, why didn't we have this before?

Reply:
Ya, this should not be required, CATCH2 will register the labels with CMake that already appear in the TEST_CASE macros, were you having problems with that?

Reply:
Ah, I didn't realize catch2 automatically labeled them all with unit_tests. I'll remove this.